sass/base/_buttons.scss

```diff
@@ -1,34 +1,53 @@
-.btn {
-  border-radius: 0.4em;
-  padding: 0.25em 1em;
+$button-color: $button-color;
```

Comment:
Very certain I'm missing something here but is this line needed? Surely we can solely use `$button-color`?